src/plumtree_broadcast.erl

```diff
@@ -645,7 +657,7 @@ send(Msg, Mod, P) ->
                                       partisan_peer_service),
     PeerServiceManager = PeerService:manager(),
     instrument_transmission(Msg, Mod),
-    PeerServiceManager:forward_message(P, ?SERVER, Msg).
+    ok = PeerServiceManager:forward_message(P, ?SERVER, Msg).
```

Comment:
it doesn't crash the caller but it does crash the `plumtree_broadcast` worker process, returning error here doesn't really help since it is being ignored on all invocations